### DISCUSSION TITLE
Monika has no face, along with other graphical glitches.

### DISCUSSION THREAD

**Discussion by theEpicjosh** [Monika-After-Story/MonikaModDev#7776] [QA - tech support + general] [answered]
![image](https://user-images.githubusercontent.com/25969231/130893665-62533171-481f-4c60-b47f-3c401e889b75.png)
Monika appears to have no face, and other graphical glitches.
This happened yesterday, as well. (I restarted Monika After Story between the two days.)

I don't know what would've caused this...

**Comment by Ulti70895** [comment]
Omg that's so creepy as she told me a scary story about a woman with no face lol.

**Comment by theEpicjosh** [comment]
Overtime, she has had more graphical glitches.
Such as her head disappeared, her clothing disappeared (including her chest), and even the room started to break down (such as the walls started to disappear).

I don't know what would've happened, but if I could get a suggestion, I would like that.

Thanks :) .

P.S: I'll try to send a photo of what this looks like tomorrow. (I'm currently at a coffee shop. My house won't have internet until tomorrow. Something's up with the router...)

**Comment by multimokia** [comment]
What are your computer's specs? Can you send the `log.txt` file? It's in the main directory for the game (the same one with `DDLC.exe` in it)

**Comment by theEpicjosh** [comment]
My computer is a Windows 10 v21H1 64-bit with
- 16 GB RAM
- Intel Core i5 4590 CPU
- and an AMD Radeon HD 7800 Series GPU

And the "log.txt" file is attached in the document:

[log.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7073048/log.txt)

**Comment by theEpicjosh** [comment]
Update 2:
![image](https://user-images.githubusercontent.com/25969231/133348423-b2045d74-1df8-4b3c-b1ef-f2c80739bebf.png)
As you can see, her clothing started to disappear as well.
Sometimes, her head started to disappear (with her face still intact), but the most common graphical glitch is still the missing face (as shown on the original post).

**Comment by ThePotatoGuy** [comment]
Couple of questions:
* Are you running this off an SSD or HDD? 
* Are you running from a temp folder?
* Are you minimizing the window?

**Comment by theEpicjosh** [comment]
> Couple of questions:
> 
> * Are you running this off an SSD or HDD?
> * Are you running from a temp folder?
> * Are you minimizing the window?

Hello. Thanks for getting back to me! :)

1. I am running Windows, as well as the game and mod on an SSD
2. I wouldn't know what running from a temp folder means. If you could elaborate, I would be happy to answer that for you. I'm running the mod off of "Doki Doki Mod Manager". My mod is located in: _%appdata%\ DokiDokiModManager\ GameData\ installs\ monika-after-story_. I doubt Doki Doki Mod Manager would affect it, because this hasn't happened when I first got the mod.
3. I minimize and keep this program in the background constantly throughout the day. Again, this hasn't affected the mod when I first got it.

Hopefully this answers some of your questions, if you need any more information, let me know.
Thanks! :)

**Comment by ThePotatoGuy** [comment, marked as answer]
minimizing can cause a bunch of problems with renpy games (or at least DDLC). You may not have experienced issues before, but it is known to cause graphical issues.
